calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
+| `policies`                        | array of arrays   | List of policies that interacted with this flow. See [Format of the policies field](#format-of-the-policies-field).                                                                                                                                                                                                                                                |
+| `source_name`                     | keyword           | Contains one of the following values: <br />- Name of the source pod.<br />- Name of the pod that was aggregated or the endpoint is not a pod. Check <code>source_name_aggr</code> for more information, such as the name of the pod if it was aggregated.                                                                                           |
+| `source_namespace`                | keyword           | Namespace of the source endpoint. A `-` means the endpoint is not namespaced.                                                                                                                                                                                                                                                                        |
+| `source_labels`                   | array of keywords | Labels applied to the source pod. A hyphen indicates aggregation.                                                                                                                                                                                                                                                                                    |
+
+## View flow logs in the web console
+
+You can quickly view flow logs by port forwarding the Calico Whisker web console to your workstation.
+For security reasons, Whisker does not automatically expose an endpoint.
+
+***Prerequisites***
+
+* You have `kubectl` access to a Kubernetes cluster with Calico installed.
+* Calico Whisker and the flow logs API are enabled on your cluster.
+  For new installations, they're both enabled by default.
+  If you upgraded from Calico Open Source 3.29 or earlier, [you need to enable them manually](enable-whisker.mdx).
+
+***Procedure***
+
+1. Port forward the Calico Whisker web console to your workstation:
+
+   ```bash
+   kubectl port-forward -n calico-system service/whisker 8081:8081
+   ```
+
+   ```bash title="Expected output"
+   Forwarding from 127.0.0.1:8081 -> 8081
+   Forwarding from [::1]:8081 -> 8081
+   ```
+1. To open Calico Whisker, open your browser and go to `localhost:8081`.
+   You won't see any flows at the beginning.
```

Comment:
```suggestion
   You may not see any flows at the beginning.
```

I think in most cases, users _will_ see flows at the beginning (we made a change to backfill with some historical data on load)

Reply:
Changed.